src/Features/Core/Portable/InlineHints/AbstractInlineParameterNameHintsService.cs

```diff
@@ -90,10 +91,15 @@ void AddHintsIfAppropriate(SyntaxNode node)
 
                     if (HintMatches(kind, literalParameters, objectCreationParameters, otherParameters))
                     {
+                        var replacementTextFunction = doubleClickToInsertHint
+                            ? InlineHintHelpers.GetReplacementTextFunction(parameter.Name)
+                            : null;
+
                         result.Add(new InlineHint(
                             new TextSpan(position, 0),
                             ImmutableArray.Create(new TaggedText(TextTags.Text, parameter.Name + ": ")),
```

Comment:
isn't the replacement text exactly the same as this string here?  why not just compute it here and just add to the hint?

Reply:
done

Reply:
confused.   you still generate `parameter.Name + ": "` here.

Reply:
right, for C# and VB we generated "parameter:" for the hint itself, which I thought should remain the same. It only matters to generate the parameter:= for VB when you double click and replace the hint, which is why I added the "inlineHintText" var on line 94.

Reply:
interesting.  it's arguable that the VB hint is wrong and should be `foo:=` to indicate the actual VB implied text... can you try that out and show me what it looks like?  if it's not hideous, we could mvoe to that.  If it's awful, then your solution is a good workaround.